bundles/org.openhab.binding.logreader/src/main/java/org/openhab/binding/logreader/internal/handler/LogHandler.java

```diff
@@ -156,7 +156,7 @@ private void shutdown() {
     @Override
     public void fileNotFound() {
         final String msg = String.format("Log file '%s' does not exist", configuration.filePath);
-        logger.debug(msg);
+        logger.debug("{}", msg);
```

Comment:
Isn't this a warning :-)

Reply:
IMO this should be removed completely. The same message is set in the thing status.